src/CoreBluetooth/Enums.cs

```diff
@@ -23,6 +24,17 @@ public enum CBManagerState : nint {
 		PoweredOff,
 		PoweredOn
 	}
+#else
```

Comment:
Yes. The next two enums "steal" the values.

Good idea on sharing the guts of the enum.